Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
-gem 'resque-unique_in_queue'
+gem 'resque-unique_in_queue', github: 'teamintricately/resque-unique_in_queue', ref: 'fdbfe41' # Ensure that only one job per signature can be queued at a time
```

Comment:
Update ref once [this PR](https://github.com/teamintricately/resque-unique_in_queue/pull/2) gets merged.